kahuna/public/js/search/structured-query/query-suggestions.js

```diff
@@ -60,6 +61,10 @@ const subjects = [
     'weather'
 ];
 
+const isSearch = [
+  'staff'
```

Comment:
this should probably be returned by an endpoint, to be more scalable...

Reply:
Agreed although at least it's isolated here. If we do make an API for it I suggest we follow the data model @itsibitzi defined for the new PFI search component: https://github.com/guardian/threads/tree/master/src/components/search. That way switching it in will be easier